cmd/cosign/main.go

```diff
@@ -43,6 +43,9 @@ func main() {
 		} else if strings.HasPrefix(arg, "-") {
 			// Handle -output, convert to --output
 			newArg := fmt.Sprintf("-%s", arg)
+			if newArg == "--version" {
+				newArg = "version"
```

Comment:
sounds good, but the message that outputs is not correct, maybe we can change the message for this case?
because it says

````
$ ./cosign -version
WARNING: the flag -version is deprecated and will be removed in a future release. Please use the flag version.
```

is to use the `command` and not the `flag`